lib/src/processors/processor.dart

```diff
@@ -189,9 +191,14 @@ class Processor extends AbstractProcessor<void> {
             K.androidSrcPath,
             config: flavorizr,
           ),
-      'android:icons': () => AndroidIconsProcessor(
-            config: flavorizr,
-          ),
+      'android:icons': () => QueueProcessor([
+            AndroidIconsProcessor(
+              config: flavorizr,
+            ),
+            AndroidAdaptiveIconsProcessor(
+              config: flavorizr,
+            ),
+          ], config: flavorizr),
```

Comment:
Instead of having a QueueProcessor here, you can collapse the `AndroidAdaptiveIconProcessor` behaviour into the `AndroidIconsProcessor` using the spread operator over the super constructor:

``` dart
...config.androidFlavors
     .where((_, flavor) =>
         flavor.android!.adaptiveIcon!['foreground'] != null &&
         flavor.android!.adaptiveIcon!['background'] != null)
     .map((flavorName, flavor) => MapEntry(
           flavorName,
           QueueProcessor([
             AndroidAdaptiveIconXmlProcessor(flavorName,
                 config: config),
             AndroidAdaptiveIconProcessor(
               flavor.android!.adaptiveIcon!['foreground'] ?? '',
               flavor.android!.adaptiveIcon!['background'] ?? '',
               flavorName,
               config: config,
             ),
           ], config: config),
         ))
     .values,
```

Also, I think it's better to extend classes starting from QueueProcessors instead of using them in this way IMHO.